### ISSUE TITLE
[🐛] 🔥 Gradle play-services dependencies - upgrade

### ISSUE THREAD

**Issue by Gregoirevda** [invertase/react-native-firebase#6046]
## Issue

Google PlayStore gives a warning 
```
play-services-tasks v18.0.0 has known issues that can lead to runtime NPEs (NullPointerException) when handling Task<Void>
results in Kotlin code. We strongly recommend that you avoid using this version. This also impacts the SDKs listed in the December 
9, 2021 release, because they all depend on play-services-tasks. A fix for this issue is included in play-services-tasks v18.0.1. See
 https://developers.google.com/android/guides/releases#december_16_2021
```

https://developers.google.com/android/guides/releases#december_16_2021

**Comment by Gregoirevda** [comment]
Could you provide a new version which contains version ^18.0.1 for 
play-services-base and play-services-tasks please?

**Comment by mikehardy** [comment]
You have skipped the issue template where we ask you for detailed information that helps us troubleshoot your problem so I can't tell whether or not you have already updated to the version we provide that has the update.

https://stackoverflow.com/help/how-to-ask

https://firebase.google.com/support/release-notes/android#bom_v29-0-34
https://github.com/invertase/react-native-firebase/blob/main/CHANGELOG.md#1410-2021-12-18
https://github.com/invertase/react-native-firebase/commit/051f4a66d64db42f1c615580e185eaf00660fbc1

**Comment by Gregoirevda** [comment]
Fixed in 14.3.0 indeed

**Comment by Jbug1984** [comment]
Thank u if ur helping me I have no one
